### PR TITLE
Create v.2.48.0.md

### PR DESCRIPTION
**Pull request by ViktarStarastsenka**
Please do not merge them for now

### CODE REVIEW

File: content/develop/connect/insight/release-notes/v.2.48.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+Title: RedisInsight v2.48.0, April 2024
```
Comment (dwdougherty) [inline content/develop/connect/insight/release-notes/v.2.48.0.md:2]:
```suggestion
Title: Redis Insight v2.48.0, April 2024
```

File: content/develop/connect/insight/release-notes/v.2.48.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+linkTitle: v2.48.0 (April 2024)
+date: 2024-04-10 00:00:00 +0000
+description: RedisInsight v2.48
```
Comment (dwdougherty) [inline content/develop/connect/insight/release-notes/v.2.48.0.md:5]:
```suggestion
description: Redis Insight v2.48
```

File: content/develop/connect/insight/release-notes/v.2.48.0.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+weight: 1
+aliases: /ri/release-notes/v2.48.0/
+         /ri/release-notes/v2.48.0.md
```
Comment (dwdougherty) [inline content/develop/connect/insight/release-notes/v.2.48.0.md:8]: Remove lines 7 and 8.

File: content/develop/connect/insight/release-notes/v.2.48.0.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+---
+## 2.48 (April 2024)
+This is the General Availability (GA) release of RedisInsight 2.48.
```
Comment (dwdougherty) [inline content/develop/connect/insight/release-notes/v.2.48.0.md:11]:
```suggestion
This is the General Availability (GA) release of Redis Insight 2.48.
```

File: content/develop/connect/insight/release-notes/v.2.48.0.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+- New look, still fast. We've refreshed our Redis Insight app to align with our new brand look
+- Jumpstart your Redis journey by uploading sample data for empty databases
+- Support for the static proxy subpath to run Redis Insight on Docker behind a gateway
+
+### Details
+
+**Features and improvements**
+- [#3233](https://github.com/RedisInsight/RedisInsight/pull/3233) New look, still fast. We've refreshed our Redis Insight app to align with our new brand look.
+- [#3224](https://github.com/RedisInsight/RedisInsight/pull/3224) Jumpstart your Redis journey by uploading sample data with JSON and basic data structures for empty databases. To upload the sample data, navigate to the Browser screen for your empty database and initiate the upload process with just a click.
+- [#2711](https://github.com/RedisInsight/RedisInsight/pull/2711) Support for the static proxy subpath to run Redis Insight on Docker behind a gateway. Use the `RIPROXYPATH` environment variable to configure the subpath proxy path.
```
Comment (dwdougherty) [inline content/develop/connect/insight/release-notes/v.2.48.0.md:23]: Use the new repo path: .../Redis-Insight/Redis-Insight/...

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Needs some work; see comments.

File: content/develop/connect/insight/release-notes/v.2.48.0.md (hunk 3 above)
Comment (ViktarStarastsenka) [reply content/develop/connect/insight/release-notes/v.2.48.0.md]: removed

File: content/develop/connect/insight/release-notes/v.2.48.0.md (hunk 5 above)
Comment (ViktarStarastsenka) [reply content/develop/connect/insight/release-notes/v.2.48.0.md:22]: the repo will stay the same - RedisInsight/RedisInsight

**Review by dwdougherty — APPROVED** [review]
LGTM

**Pull request merged**